### PR TITLE
4 db schema

### LINKED ISSUE #4: Database schema
- [x] Create postgres database
  - [x] dev
  - [x] prod
  - [x] test
- [x] Migrate tables (see attached schema)
  - [x] dev 
  - [x] prod
  - [x] test
notarynow-dbschema-2-15-20

### PR DESCRIPTION
**Pull request by BabsLabs**
Fixes #4 

Type of change
 🐣 New feature (non-breaking change which adds functionality)
 📃 This change requires a documentation update

Summary of changes
- Add models for the database
- Add migrations for the database

How Has This Been Tested?
No...

Checklist:
 ✅ I have performed a self-review of my own code
 🚫 My changes generate no new warnings

### CODE REVIEW

File: notary_now_api/models.py [hunk 1]
```diff
@@ -13,3 +13,26 @@ class User(models.Model):
     active = models.BooleanField(default=True)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+class Location(models.Model):
```
Comment (lrs8810) [inline notary_now_api/models.py:17]: ✅ Add models for the database

**Review by davidlopez — APPROVED** [review]
Migrations are looking good

File: notary_now_api/migrations/0002_appointment_language_location.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.0.3 on 2020-02-15 21:08
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```
Comment (lrs8810) [inline notary_now_api/migrations/0002_appointment_language_location.py:7]: ✅ Add migrations for the database

File: notary_now_api/migrations/0003_auto_20200215_2109.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.0.3 on 2020-02-15 21:09
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Comment (lrs8810) [inline notary_now_api/migrations/0003_auto_20200215_2109.py:6]: ✅ Add migrations for the database
Interesting how Django only needs the name `user` and not `user_id`

**Review by lrs8810 — APPROVED** [review]
Looks great!

**Pull request merged**